docs/permissions.md

```diff
@@ -372,6 +372,20 @@ Accept: application/vnd.api+json
             "codes": {
                 "jane": "Yohyoo8BHahh1lie"
             }
+        },
+        "cozyMetadata": {
+            "doctypeVersion": 1,
+            "metadataVersion": 1,
+            "createdAt": "2019-05-14T11:01:41.687737935+02:00",
+            "createdByApp": "drive",
+            "updatedAt": "2019-05-14T12:00:37.372193145+02:00",
+            "updatedByApps": [
+                {
+                    "slug": "drive",
+                    "date": "2019-05-14T12:00:37.37219352+02:00",
+                    "version": "1.1.0"
+                }
+            ]
```

Comment:
I think it would be safer to not let an application sets the `updatedByApps`. It should be the stack that keep a trace of which apps have made changes to a permission document.